raspbian/applications/rpi-update.md

```diff
@@ -0,0 +1,37 @@
+# rpi-update
+
+`rpi-update` is a command line application that will update your Raspbian kernel and Videocore firmware to the latest pre-release versions.
+
+**WARNING: Pre-release versions of software are not guaranteed to work. You should not use `rpi-update` on any system unless recommended to do so by a Raspberry Pi engineer. It may leave your system unreliable or even completely broken. It should not be used as part of any regular update process**
+
+The `rpi-update` script is supplied by a third party, `Hexxeh`, and also supported by Raspberry Pi engineers. The script source can be found on Github at https://github.com/Hexxeh/rpi-update
+
+## What it does
+
+`rpi-update` will download the latest pre-release version of the linux kernel, it's matching modules, device tree settings, along with the latest version of the Videocore firmware. It will then install these files to the boot partition of the SD card, overwriting any previous versions. 
```

Comment:
"device tree settings" -> "device tree files" ?
@pelwell 

IIRC it also includes the userland libraries and binaries? (which obviously go on the root partition rather than the boot partition).
Just to be pedantic, the kernel modules also go on the root partition rather than the boot partition :stuck_out_tongue_winking_eye:

Reply:
"files" is better, but I don't think you needed me to tell you that.